### ISSUE TITLE
Mobs Issue

### ISSUE THREAD

**Issue by Rhbarber** [casperge/RealisticSeasons-issues#209]
We recently had a season change from Summer to Fall. All pandas were wiped upon this change, even some used by players and which were inside fences, is this intended? Does it unspawn all season-specific mobs when changing from one to another?

**Comment by Killer15CZE** [comment]
Hello, we have problems with monsters and animals... We have a time set like in the real world and mobs do not spawn after 14:00 until the server restarts. After restarting, everything runs without errors for a few hours, then the same error again. I didn't find the error in the console.... The current season is autumn. Don't know what to do about it?

**Comment by casperge** [comment]
> We recently had a season change from Summer to Fall. All pandas were wiped upon this change, even some used by players and which were inside fences, is this intended? Does it unspawn all season-specific mobs when changing from one to another?

I wipe mobs unless they are nametagged or fed. Sometimes season mobs spawn in already existing farms with fences, so that's why I don't directly check for that. Do you know if players nametagged their mobs?

**Comment by casperge** [comment]
> Hello, we have problems with monsters and animals... We have a time set like in the real world and mobs do not spawn after 14:00 until the server restarts. After restarting, everything runs without errors for a few hours, then the same error again. I didn't find the error in the console.... The current season is autumn. Don't know what to do about it?

Hello, can you open a separate post on this GitHub repository about it? Otherwise, things get unorganized.

**Comment by Rhbarber** [comment]
> > We recently had a season change from Summer to Fall. All pandas were wiped upon this change, even some used by players and which were inside fences, is this intended? Does it unspawn all season-specific mobs when changing from one to another?
> 
> I wipe mobs unless they are nametagged or fed. Sometimes season mobs spawn in already existing farms with fences, so that's why I don't directly check for that. Do you know if players nametagged their mobs?

No, they weren't, and I can't really blame you on the fence-checking, I was just wondering before looking into any other issues, thanks for getting back to us!
